Ably.podspec

```diff
@@ -24,4 +24,5 @@ Pod::Spec.new do |s|
   s.dependency 'msgpack', '0.3.1'
   s.dependency 'ULID', '1.1.0'
   s.dependency 'SAMKeychain', '1.5.3'
+  s.dependency 'DeltaCodec', '1.1.0'
```

Comment:
Hi @ricardopereira ... I've [raised an issue](https://github.com/ably/delta-codec-cocoa/issues/8) against `delta-codec-cocoa` for you to rename this. Once you've done so then could you please use that new pod in this codebase. Happy for you to commit straight against this integration branch. Thanks.